openedx/features/wikimedia_features/wikimedia_general/signals.py

```diff
@@ -119,18 +119,21 @@ def send_new_post_email_notification_to_instructors(sender, user, post,**kwargs)
 
     post_id = post.course_id
     course_key = CourseKey.from_string(post.course_id)
-    
+    data = post.to_dict()
+    get_courseware_info(data, user, current_site, course_key)
```

Comment:
Name of this function is a bit misleading since it doesn't return anything how about renaming it to something like `add_courseware_info`.

Reply:
function renamed